samples/server/petstore/springboot/pom.xml

```diff
@@ -1,10 +1,10 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.openapitools</groupId>
-    <artifactId>springboot</artifactId>
+    <artifactId>openapi-spring</artifactId>
     <packaging>jar</packaging>
-    <name>springboot</name>
-    <version>1.0.0-SNAPSHOT</version>
+    <name>openapi-spring</name>
+    <version>1.0.0</version>
```

Comment:
The change results in the following errors reported by the CI:
```
Running node 1 to test 'samples.circleci' defined in pom.xml ...
openjdk version "1.8.0_141"
OpenJDK Runtime Environment (build 1.8.0_141-8u141-b15-3~14.04-b15)
OpenJDK 64-Bit Server VM (build 25.141-b15, mixed mode)
[ERROR] Project 'org.openapitools:openapi-spring:1.0.0' is duplicated in the reactor -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DuplicateProjectException
```

Reply:
This was auto generated. Did an undo manually now (don't know if this is the solution)

Reply:
Did you remove the file before regenerating the samples? (you shouldn't need to remove the samples manually)

Reply:
No, I only modified the two mustache templates manually. The rest of the modifications must have been done by tests/regeneratiing the samples or probably the development environment.

Reply:
Could the problem be, that I'm using Windows and there is no appropriate script to update the petstore sample for windows? Which scripts do I need to execute for Windows and my changes in the JavaSpring template such that the ci tests do not fail any more?

Reply:
ah.. likely. let me try to update the samples for you.

Reply:
This would be nice.